src/HotChocolate/Spatial/src/Types.Spatial/CRS/GeoJSONCoordinateReferenceSystem.cs

```diff
@@ -0,0 +1,21 @@
+namespace Types.Spatial
```

Comment:
- [ ] Fix namespaces to HotChocolate.Types.Spatial

Reply:
It looks like you already made this change. Are you suggesting that the optional CRS values be moved into the Spatial namespace? I thought since they are an "extension" that we would keep them out?

Reply:
Since your review comment is no longer valid i'm going to resolve this. bring up a new issue if you want for the namespace name choice